website/docs/cloud-docs/agents/changelog.mdx

```diff
@@ -1,12 +1,12 @@
 ---
-page_title: Changelog - Terraform Cloud Agents - Terraform Cloud and Terraform Enterprise
+page_title: Changelog - HCP Terraform Agents - HCP Terraform and Terraform Enterprise
```

Comment:
This page is actually auto-generated from the CHANGELOG in the tfc-agent's source code. There is a template within that repo that you'll need to update, otherwise this will be undone the next time there is a tfc-agent release.

Reply:
Ah, gotcha! Thank you for that context. I reverted the file. Just to explicitly confirm: your team is going to handle this in the next release I presume?